ouroboros-consensus/src/Ouroboros/Consensus/Storage/LedgerDB/HD/ToStore.hs

```diff
@@ -0,0 +1,3 @@
+module Ouroboros.Consensus.Storage.LedgerDB.HD.ToStore (ToStoreKind (UTxO)) where
```

Comment:
Seems unnecessary to have a separate module just for this datatype, but it prevents cyclic dependencies